example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name="${applicationName}"
```

Comment:
The changes followed the migration guide https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects